src/chrome/content/rules/Migros-Bank.xml

```diff
@@ -1,42 +1,35 @@
 <!--
 	For other Migros coverage, see Migros.xml.
-
-	Mismatch:
-		- blog.bancamigros.ch
-		- p2p.bancamigros.ch
-		- blog.banquemigros.ch
-		- p2p.banquemigros.ch
```

Comment:
Are you involved? Then you should have a look at preloaded HSTS:
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#hsts-preloaded-rules
https://hstspreload.org/